unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Utils/BuilderInWorldUtils.cs

```diff
@@ -111,7 +146,7 @@ public static string ConvertEntityToJSON(DecentralandEntity entity)
             {
                 EntityData.TransformComponent entityComponentModel = new EntityData.TransformComponent();
 
-                entityComponentModel.position = Environment.i.worldState.ConvertUnityToScenePosition(entity.gameObject.transform.position, entity.scene);
+                entityComponentModel.position = DCL.Environment.i.sceneController.ConvertUnityToScenePosition(entity.gameObject.transform.position, entity.scene);
```

Comment:
This should be referencing `worldState`. I'm missing something here?

Reply:
WorldState is new for me, I can change it without problems, What is the purpose of WorldState?

Reply:
Also, if we should reference to `worldState`, `WorldState `class should have the inversed method? `ConvertScenePositionToUnityPosition`